### ISSUE TITLE
Get the arrow to the element instead of the beginning of the box

### ISSUE THREAD

**Issue by Daltonstudio** [juho-jaakkola/elgg-tour#7]
The left side of the tourstop container is aligned to the element, but on small elements (like an icon) this causes the arrow to be pointing to 'nothing' or a wrong element. Adding a margin-left to .joyride-tip-guide of -27px and enlarging the width of .joyride-tip-guide with 27px could solve it, not sure if it's the proper way though.
![arrow alignment](http://driesdk.be/junk/align.jpg)

**Comment by juho-jaakkola** [comment]
What selector are you using in the example?

**Comment by Daltonstudio** [comment]
Sorry, the mail icon, but it's the same for all, the left border of the wrapper aligns to the left border of the target element, which could cause misunderstandings because the arrow might be more to the right then the actual target is.

**Comment by juho-jaakkola** [comment]
I've noticed the same, but I'm not yet sure if this should be considered a bug or user error.

In the default theme a good workaround is to use the whole `<li>` element instead of the icon. Does that work in your theme?

**Comment by Daltonstudio** [comment]
No, not really, but I see what you mean, and I see it's in my case a user error and it has to do with the styling of my icons, so I could solve it there. Consider it a user error then :)
![align](http://driesdk.be/junk/align2.jpg)
